compiler_gym/wrappers/core.py

```diff
@@ -371,3 +376,70 @@ def multistep(
     def convert_reward(self, reward):
         """Translate a reward to the new space."""
         raise NotImplementedError
+
+
+class ConversionWrapperEnv(CompilerEnvWrapper):
```

Comment:
I'm not sure about this. I like that the wrappers in base gym operate only on action/observation/reward at a time. Could you add the space conversion methods to the relevant wrapper and use them instead? E.g. merge `convert_observation_space` into `ObservationWrapper`. You could then have:

```py
env = MlirObservationWrapper(MlirActionWrapper(MlirRewardWrapper(env)))
```

Reply:
Done.